src/utils/isAddrEqual/index.ts

```diff
@@ -2,8 +2,8 @@ import { Address } from '../../generated/graphql'
 
 export const isAddrEqual = (address: Address, addressStr: string): boolean => {
   const noNetworkAddr = addressStr.slice(1)
-  const noNetworkID = address.id?.slice(1)
-  const noNetworkRobust = address.robust?.slice(1)
+  const noNetworkID = address?.id?.slice(1)
+  const noNetworkRobust = address?.robust?.slice(1)
```

Comment:
Just a safe guard in case the server returns undefined since either address is now technically "optional", even though it should always at least return an empty string